.github/workflows/build-windows-package.yml

```diff
@@ -120,7 +120,7 @@ jobs:
             "7.4")
               COMPILER="vc15"
               ;;
-            "8.0" | "8.1" | "8.2")
+            "8.0" | "8.1" | "8.2" | "8.3")
```

Comment:
Thanks, it looks like this was missed in 2d0ccfc1e84b04542e14416b5a181be4ae740fee for #1655.